packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/configure-packages/react-native/configuremid.md

```diff
@@ -0,0 +1,13 @@
+Call `createConfig` function to create a configured client on the native modules. Resolves true if it successfully configures a client.
```

Comment:
"the" `createConfig` function

Call the `createConfig` function to create a configured client on the native modules. Resolves true if it successfully configures a client.